composer.json

```diff
@@ -18,18 +18,19 @@
     "require-dev": {
         "doctrine/coding-standard": "^4.0",
         "phpstan/phpstan": "^0.9.2",
+        "phpstan/phpstan-phpunit": "^0.9.4",
```

Comment:
I believe we need to import it to `phpstan.neon`. We have many `assertEquals`, and green build on Travis :(

Reply:
I changed all `self::assertEquals` calls to `self::assertSame`